File: apps/docs/src/code/python/basics/download_file.py
from e2b import Sandbox

session = Sandbox.create(id="Nodejs")

file_in_bytes = session.download_file('path/to/remote/file/inside/sandbox') # $HighlightLine
# Save file to local filesystem
with open('path/to/local/file', 'wb') as f: # $HighlightLine
  f.write(file_in_bytes) # $HighlightLine

session.close()

File: apps/docs/src/code/js/basics/download_file.js
import { Sandbox } from '@e2b/sdk'
import fs from 'node:fs'

const sandbox = await Sandbox.create({id: 'base'})

const buffer = await sandbox.downloadFile('path/to/remote/file/inside/sandbox') // $HighlightLine
// Save file to local filesystem
fs.writeFileSync('path/to/local/file', buffer)

await sandbox.close()
